api/office_structure/schema.py

```diff
@@ -61,6 +61,20 @@ def mutate(self, info, id):
         return DeleteNode(node=node)
 
 
+class Query(graphene.ObjectType):
+    get_node = graphene.Field(
```

Comment:
Use variable names that are not verbs in nature

Reply:
@Bonifase This has been worked on